build.zig.zon

```diff
@@ -5,7 +5,7 @@
 
     .fingerprint = 0x76501fb842f52025, // Changing this has security and trust implications.
 
-    .minimum_zig_version = "0.15.0-dev.355+206bd1ced",
+    .minimum_zig_version = "0.15.0-dev.12486+a843be44a",
```

Comment:
The latest version available for download from ziglang.org is `0.15.0-dev.451+a843be44a`, this is ~12k commits ahead of that.

Reply:
I guess we have a long release cycle ahead of us. 👀

Reply:
I compiled zig manually from source code

Reply:
That still doesn't get you a version like that. Current Zig master as of this comment is `0.15.0-dev.451+a843be44a`.

Reply:
You're building your fork of Zig in which the latest tagged version is 0.10.1.